pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -188,9 +182,6 @@ func (r *OnlineUpgradeReconciler) Reconcile(ctx context.Context, _ *ctrl.Request
 		// replica group B.
 		r.postSandboxSubclustersMsg,
 		r.sandboxReplicaGroupB,
```

Comment:
Could we uncomment the lines 197 and 198:
// r.postPrepareReplicationMsg,
// r.prepareReplication

Also remove //nolint:unused for those functions.